devicetypes/aamann/enlighten-envoy-local.src/enlighten-envoy-local.groovy

```diff
@@ -296,9 +302,10 @@ def ping() {
 
 def startPoll() {
 	unschedule()
-	// Schedule 2 minute polling
+	def wantMin = 6       // Schedule N minute polling Integer < 60
```

Comment:
On a normal day I get three or four polls in a row where the `parse()` function does not get invoked - does this not happen for you?

I reworked your pull request and added a new setting - I guess I'll set the default to 5 minutes.

Thanks for your suggestions - especially the `updated()` issues has always bothered me!